ssh_automated_tests/core/connection.py

```diff
@@ -10,13 +10,17 @@ class Connection(ABC):
     def connect(self):
         pass
 
+    @abstractmethod
+    def disconnect(self):
+        pass
 
-class SshConnection:
+
+class SshConnection(Connection):
```

Comment:
I'm worried that `SshConnection` has more public methods then declared in `Connection`. And for me, it means that there is a problem with the design. I just suggested my vision how to solve this.